kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/extensions/TestKubeKmsFacadeInvocationContextProvider.java

```diff
@@ -61,7 +65,18 @@ public List<Extension> getAdditionalExtensions() {
                             return kmsFacade;
                         }
                     },
-                    (AfterEachCallback) extensionContext -> kmsFacade.stop());
+                    (AfterEachCallback) extensionContext -> {
+                        try {
+                            Optional<Throwable> exception = extensionContext.getExecutionException();
+                            exception.filter(t -> !t.getClass().getSimpleName().equals("AssumptionViolatedException")).ifPresent(e -> {
+                                DeploymentUtils.collectClusterInfo(String.join(",", Vault.VAULT_DEFAULT_NAMESPACE, LocalStack.LOCALSTACK_DEFAULT_NAMESPACE),
```

Comment:
This is leaking knowledge of how the tests work into the tests.  It won't scale.  We'll be adding Apicurio Registry soon and there will be other operands.   It would be better if we could avoid that.  

How about add a kube label (`kroxylicious-test-operand`) to all the Vault and Localstack namespaces.  Then change collectClusterInfo to additionally collect info from all namespaces with the label.